packages/gimbal/yarn-error.log

```diff
@@ -0,0 +1,146 @@
+Arguments: 
```

Comment:
Can we `git rm --cached yarn-error.log` and also add it to `.gitignore`?

Reply:
I ignored, just need to `rm` both files https://github.com/ModusCreateOrg/gimbal/commit/808c1bb11efb007b650524554b631551d3dfa852